app/code/Magento/SalesGraphQl/Model/Resolver/CustomerOrders/Query/OrderFilter.php

```diff
@@ -65,25 +65,18 @@ public function __construct(
      *
      * @param array $args
      * @param int $userId
-     * @param int $storeId
      * @return FilterGroup[]
      */
     public function createFilterGroups(
         array $args,
-        int $userId,
-        int $storeId
+        int $userId
```

Comment:
I'm not sure that removing argument from public method is a good idea.
I believe that we can make `$storeId` nullable or provide new method and deprecate current that more preferable